### PR TITLE
Add handling of private aggregation paggResponse field in response

### PR DESCRIPTION
**Pull request by qingxinwu**

### CODE REVIEW

**Comment by qingxinwu** [comment]
@brusshamilton PTAL, thanks!

File: draft-ietf-bidding-and-auction-services.md [hunk 1]
```diff
@@ -391,10 +391,11 @@ the {{framing}} header.
 This section describes how the client MAY form and serialize request messages
 in order to communicate with the Bidding and Auction services.
 
-This algorithm takes as input all of the `relevant interest groups`, and a config
-consisting of the `publisher`,
+This algorithm takes as input all of the `relevant interest groups`, a config
+consisting of the `publisher`, a map of from (origin, string) tuple to origin
+`ig pagg coordinators`,
 an optional `desired total size`, an optional list of `interest group owners` to
-include each with an optional `desired size`, and the [HPKE] `public key` and
```
Comment (brusshamilton) [inline draft-ietf-bidding-and-auction-services.md:397]: The extra "and" was intentional here, as a way of grouping the key with its key ID. We could change the second "and" to "with" if you think that would be more clear.

File: draft-ietf-bidding-and-auction-services.md [hunk 2]
```diff
@@ -391,10 +391,11 @@ the {{framing}} header.
 This section describes how the client MAY form and serialize request messages
 in order to communicate with the Bidding and Auction services.
 
-This algorithm takes as input all of the `relevant interest groups`, and a config
-consisting of the `publisher`,
+This algorithm takes as input all of the `relevant interest groups`, a config
+consisting of the `publisher`, a map of from (origin, string) tuple to origin
```
Comment (brusshamilton) [inline draft-ietf-bidding-and-auction-services.md:395]: I would have thought we could calculate `ig pagg coordinators` from the `relevant interest groups`. I think we'd rather do that than have it as a separate argument.

File: draft-ietf-bidding-and-auction-services.md [hunk 3]
```diff
@@ -471,7 +472,8 @@ tuple.
    larger than request.
 1. Encrypt `request` using the `public key` and its `key id` as in
    {{request-encryption}} to get the `encrypted message` and `hpke context`.
-1. Let the `request context` be the tuple (`included_groups`, `hpke context`)
+1. Let the `request context` be the tuple
+   (`included_groups`, `hpke context`, `ig pagg coordinators`).
```
Comment (brusshamilton) [inline draft-ietf-bidding-and-auction-services.md:476]: Would it make more sense to expand `included_groups` to be a list of (name, pagg coordinator) pairs? That would avoid the extra redundancy.

Comment (qingxinwu) [reply draft-ietf-bidding-and-auction-services.md:476]: Do you mean let included_groups[owner]'s value be a list of (ig, coordinator) pairs? Sure we can do that, with an extra step to combine `interest group list` and `coordinator`. If that sounds good, I can make the change.

File: draft-ietf-bidding-and-auction-services.md (hunk 2 above)
Comment (qingxinwu) [reply draft-ietf-bidding-and-auction-services.md:395]: but it seems step 10.4 of https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#section-2.2.4 serialized everything

File: draft-ietf-bidding-and-auction-services.md (hunk 3 above)
Comment (qingxinwu) [reply draft-ietf-bidding-and-auction-services.md:476]: the field is only used by this spec in "parse response", so it needs to be available in `request context`. But if we want to add it to be part of included_groups (as what's being discussed in the other comment), then we don't need this extra field.

File: draft-ietf-bidding-and-auction-services.md (hunk 2 above)
Comment (brusshamilton) [reply draft-ietf-bidding-and-auction-services.md:395]: Ah, I didn't realize we weren't explicitly serializing it. Then I guess we do need to pass it separately.

**Review by brusshamilton — APPROVED** [review]

**Comment by qingxinwu** [comment]
@brusshamilton Made some final changes based on w3c's last minute changes. PTAL. Thanks

**Review by brusshamilton — APPROVED** [review]

**Pull request merged**